database/postgres/seedPostgresqlActor.js

```diff
@@ -31,13 +30,24 @@ const seedPostgresActor = async () => {
     `);
 
     console.log('writing to database!');
-    const copyPath = path.join(__dirname, '../actorData.csv');
+    const copyPath = path.join(__dirname, '../../actorData.csv');
     await client.query(`
       COPY ActorInfo FROM '${copyPath}' WITH (FORMAT CSV, HEADER);
     `);
-    console.log('adding auto serial index on column named "id"!');
+
+    console.log('adding auto serial index column named "id"!');
+    await client.query(`
+      ALTER TABLE actorinfo ADD COLUMN id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY;
+    `);
+
+    console.log('adding auto serial index on column named "role"!');
+    await client.query(`
+    CREATE INDEX roleindex ON actorinfo (role);
+    `);
```

Comment:
Do you need a role index anymore? I suggest making sure to remove all indexes that you don't need even from the coding script.